pkg/bosh/bpm/config.go

```diff
@@ -1,54 +1,54 @@
 package bpm
 
 import (
-	yaml "gopkg.in/yaml.v2"
+	"github.com/ghodss/yaml"
 )
 
 // Hooks from a BPM config
 type Hooks struct {
-	PreStart string `yaml:"pre_start,omitempty"`
+	PreStart string `json:"pre_start,omitempty"`
```

Comment:
This is a significant change. Assuming that `"github.com/ghodss/yaml"` needs json tags, you can still keep the yaml tags.

```
PreStart string `json:"pre_start,omitempty" yaml:"pre_start,omitempty"`
```